packages/components/src/components/icon/icons.js

```diff
@@ -291,6 +291,7 @@ import './common/ic-boleto-d24-voucher-light.svg';
 import './common/ic-bot-builder-arrow.svg';
 import './common/ic-bot-builder-tab-icon.svg';
 import './common/ic-bot-builder.svg';
+import './common/ic-bot-stop.svg';
```

Comment:
Is this needed ? 👀

Reply:
I don't think this is needed, @shontzu-deriv can you revert this file change!